ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -214,56 +214,60 @@ function AuthProviders(): ReactElement {
                     }
                 />
             )}
-            <PageSection>
-                <Alert
-                    isInline
-                    variant="info"
-                    title="Consider using short-lived tokens for machine-to-machine communications
+            {isList && (
+                <PageSection>
```

Comment:
Aha, that was a lapse in my review to miss `isList` conditional rendering.

Here is an alternative for your consideration. Add `variant="light"` prop and move this `PageSection` element:
* follow `AccessControlHeader` and `AccessControlActionBar` which already have `isList` condition
* precede end fragment `</>` at current line 206

![Alert](https://github.com/stackrox/stackrox/assets/11862657/2a77164c-1a3d-4839-8aa7-fdb15b9d6bb9)

Reply:
Indentation change causes one line wrap, so if your IDE does not display lint errors, in ui/apps/platform folder

```sh
yarn lint:fix
```